src/components/common/button/Button.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { CSSProperties } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+const Container = styled.div<{ size: ButtonSize; type: ButtonType }>`
```

Comment:
웹 표준과 웹 접근성을 위해 `button`요소를 적용하면 더 좋을 것 같습니다!

```suggestion
const Container = styled.button<{ size: ButtonSize; type: ButtonType }>`
```